### PR TITLE
PDM-2154 added report and displayId methods

### PR DESCRIPTION
**Pull request by dineshshenoy**

### CODE REVIEW

File: socotratools/socotratools/client.py [hunk 1]
```diff
@@ -284,3 +284,17 @@ def preview_endorsement_price(self, policy_locator, endorsement_name,
                 }
 
         return self.__post("/policy/" + policy_locator + "/previewEndorsementPrice", data)
+
+    def gen_report(self, report_name, start_timestamp, end_timestamp):
+        data = {'startTimestamp': start_timestamp,
+                'endTimestamp': end_timestamp
```
Comment (rahul-x-verma) [inline socotratools/socotratools/client.py]: This won't work for the unearned premium report

File: socotratools/socotratools/client.py [hunk 2]
```diff
@@ -284,3 +284,17 @@ def preview_endorsement_price(self, policy_locator, endorsement_name,
                 }
 
         return self.__post("/policy/" + policy_locator + "/previewEndorsementPrice", data)
+
+    def gen_report(self, report_name, start_timestamp, end_timestamp):
```
Comment (rahul-x-verma) [inline socotratools/socotratools/client.py]: Our back end convention is to not use abbreviations, but I'll defer to your judgment here

File: socotratools/VERSION [hunk 3]
```diff
@@ -1 +1 @@
-1.3.3
+1.3.4
```
Comment (rahul-x-verma) [inline socotratools/VERSION]: Should this change be here?

Comment (dineshshenoy) [reply socotratools/VERSION]: Yes - updating public version as I iterate

File: socotratools/socotratools/client.py (hunk 2 above)
Comment (dineshshenoy) [reply socotratools/socotratools/client.py]: Fixed.

File: socotratools/socotratools/client.py (hunk 1 above)
Comment (dineshshenoy) [reply socotratools/socotratools/client.py]: Fixed.

**Comment by dineshshenoy** [comment]
@rahul-x-verma  - can you take one more look?

**Pull request merged**